cluster-provision/k8s/1.17/provision.sh

```diff
@@ -3,11 +3,10 @@
 set -ex
 
 # Configure IPv6 DHCP
-nmcli connection modify "System eth0" ipv6.method dhcp
-nmcli connection modify "System eth0" ipv6.may-fail no
-nmcli connection modify "System eth0" connection.autoconnect yes
-nmcli connection modify "System eth0" ipv6.addr-gen-mode eui64
-nmcli connection down "System eth0" && sudo nmcli connection up "System eth0"
+nmcli connection modify "System eth0" \
+   ipv6.method auto \
+   ipv6.addr-gen-mode eui64
+nmcli connection up "System eth0"
```

Comment:
Because it is in provisioning stage, this is not needed.
Please correct me if I am wrong.

Reply:
for example when i tried in the previous config of nmcli just reload, it didnt work
i believe we need it, i can verify if needed

Reply:
Removing it cause the 2nd node to not get ipv6 fd00::102 in cluster-up.

Moreover for the provider of ipv6 we would need it, because the provision would create k8s ipv6 cluster,
unless we will do a split, that just the cluster-up would create ipv6 cluster,
and the provision would create ipv4, which is a possibility.

Reply:
Maybe we are not talking about the same thing?
`nmcli connection up "System eth0"` is activating the NM profile you modified. But as the provisioning is over, this VM does not need this config now, it needs it when it comes up later at `cluster-up`.
What am I missing here?

Reply:
i just tried to remove that exact line `nmcli connection up "System eth0"`
maybe there is an asymetric between nodes.sh and node01.sh or other reason, since its generates according the mac, but without it, node02 and only it doesnt get the fd00::102 address.

Reply:
Maybe this is not running really on the provision step? You can just check that the modification is there when the VM starts.

Reply:
this is the script that runs in the provision step
we can check everything needed in the VM indeed after it starts at the end of cluster-up

Reply:
Or maybe the profile is not saved without the `up` command? Makes no sense to me.